node/_tools/config.json

```diff
@@ -33,6 +38,7 @@
       "test-fs-append-file.js",
       "test-fs-chmod-mask.js",
       "test-fs-chmod.js",
+      "test-fs-read-stream-pos.js",
```

Comment:
I'm thinking we should just not include this test for the time being. We can revisit it when the underlying issues are addressed.

Reply:
I understand your intention. The test has been removed. e459b0b